src/init.c

```diff
@@ -560,8 +560,45 @@ static void hyper_flush_channel()
 	hyper_send_data_block(hyper_epoll.tty.fd, tty_buf->data, tty_buf->get);
 }
 
-void hyper_pod_destroyed(int failed)
+static void hyper_cleanup_mounts(struct hyper_pod *pod)
 {
+	int pid, pipe[2] = {-1, -1};
+	struct hyper_container *c;
+
+	if (pipe2(pipe, O_CLOEXEC) < 0) {
```

Comment:
enter_sandbox() still uses pipes. This can be changed when hyper_enter_sandbox() migrates to eventfd.